spec/classroom_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './../associations/classroom'
+
+describe 'Test classroom object' do
+  classroom = Classroom.new('A1')
+
+  it 'takes parameters and returns a Student object' do
+    expect(classroom).to be_instance_of(Classroom)
+  end
+
+  it "classroom label should be 'A1'" do
+    expect(classroom.label).to eql 'A1'
+  end
```

Comment:
- [OPTIONAL] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `add_student` and `students_name_list` class classroom method has (at least) an 'it' block 👇🏻 check your classroom.rb file

```ruby
  def add_student(student)
    @students << student unless @students.include?(student)
    student.classroom = self
  end

  def students_name_list
    @students.map(&:name)
  end
```